### ISSUE TITLE
When training soundstream, how to continue training on a model pt file that has been trained to a certain number of steps

### ISSUE THREAD

**Issue by So-Fann** [lucidrains/audiolm-pytorch#91]

**Comment by LWprogramming** [comment]
[Call `soundstream.load()`](https://github.com/lucidrains/audiolm-pytorch/blob/main/audiolm_pytorch/soundstream.py#L496) (Was fixed recently in release 0.12.3)

**Comment by smcio** [comment]
@So-Fann - as @LWprogramming said, you call `soundstream.load()` on your model checkpoint.

Concretely, then, I'm using the following in my Jupyter notebook (https://github.com/lucidrains/audiolm-pytorch/blob/main/audiolm_pytorch_demo.ipynb, seemingly authored by @LWprogramming) to resume training from a saved model checkpoint: 

```
soundstream = SoundStream(
    codebook_size = 1024,
    rq_num_quantizers = 8,
)

soundstream.load('results/soundstream.3000.pt')      // load your model checkpoint here

trainer = SoundStreamTrainer(
    soundstream,
    folder = dataset_folder,
    batch_size = 2,
    grad_accum_every = 8,         # effective batch size of 16
    data_max_length = 240000,
    save_results_every = 200,
    save_model_every = 500,
    num_train_steps = 100000
).cuda()
# NOTE: I changed num_train_steps to 9 (aka 8 + 1) from 10000 to make things go faster for demo purposes
# adjusting save_*_every variables for the same reason

trainer.train()
```

**Comment by ZYW507** [comment]
> Call [`soundstream.load()`](https://github.com/lucidrains/audiolm-pytorch/blob/main/audiolm_pytorch/soundstream.py#L496) (Was fixed recently in release 0.12.3)

Hi! Sorry it may be brought up in another issue, but is the soundstream.load() fixed already? cuz I tried to train again and get the same error message as I did yesterday #89 ?

**Comment by So-Fann** [comment]
> @So-Fann - as @LWprogramming said, you call `soundstream.load()` on your model checkpoint.
> 
> Concretely, then, I'm using the following in my Jupyter notebook (https://github.com/lucidrains/audiolm-pytorch/blob/main/audiolm_pytorch_demo.ipynb, authored by @LWprogramming, I believe) to resume training from a saved model checkpoint:
> 
> ```
> soundstream = SoundStream(
>     codebook_size = 1024,
>     rq_num_quantizers = 8,
> )
> 
> soundstream.load('results/soundstream.3000.pt')      // load your model checkpoint here
> 
> trainer = SoundStreamTrainer(
>     soundstream,
>     folder = dataset_folder,
>     batch_size = 2,
>     grad_accum_every = 8,         # effective batch size of 16
>     data_max_length = 240000,
>     save_results_every = 200,
>     save_model_every = 500,
>     num_train_steps = 100000
> ).cuda()
> # NOTE: I changed num_train_steps to 9 (aka 8 + 1) from 10000 to make things go faster for demo purposes
> # adjusting save_*_every variables for the same reason
> 
> trainer.train()
> ```

Thank you so much!!
